ipywidgets/widgets/widget.py

```diff
@@ -483,7 +483,7 @@ def _compare(self, a, b):
         else:
             return a == b
 
-    def _compare_buffers(self, a, b, key):
+    def _buffer_list_equal(self, a, b):
```

Comment:
Can you elaborate on this being an extension point? I don't understand how you're thinking of this as an extension point.